src/store/TodosActions.js

```diff
@@ -10,4 +10,65 @@ export function removeTodo (index) {
     type: 'REMOVE_TODO',
     index: index,
   }
+}
+
+export function showTodosSuccess (todos){
+  return {
+    type: 'SHOW_TODOS',
+    todos: todos,
+  }
+}
+
+export const addTodo = (text) => async (dispatch) => {
+  const URL = 'http://localhost:3000/addTodo';
+  const body = { todo: { text } }
+  const options = {
+    method: 'POST',
+    body: JSON.stringify(body),
+    headers: { 'Content-Type': 'application/json' }
+  }
+
+  const response = await fetch(URL, options);
+  const dataJSON = await response.json();
+
+  if(response.ok){
+    return dispatch(addTodoSuccess(dataJSON.text));
+  }
+}
+
+export const removingTodo = (text) => async (dispatch) => {
+  const URL = 'http://localhost:3000/removeTodo';
+  const body = { todo: { text } }
+  const options = {
+    method: 'DELETE',
+    body: JSON.stringify(body),
+    headers: { 'Content-Type': 'application/json' }
+  }
+
+  const response = await fetch(URL, options);
+  const dataJSON = await response.json();
+
+  if(response.ok){
+    return dispatch(removeTodoSuccess(dataJSON.text));
+  }
+}
```

Comment:
Numa API o endpoint de DELETE não recebe nenhum body, o id é passado no url
```suggestion
export const removingTodo = (id) => async (dispatch) => {
  const URL = `http://localhost:3000/removeTodo/${id}`;

  const options = {
    method: 'DELETE'
  }

  const response = await fetch(URL, options);
  const dataJSON = await response.json();

  if (response.ok) {
    return dispatch(removeTodoSuccess(dataJSON.text));
  }
}
```